src/javascript/app/pages/trade/not-available.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Text, Button } from '@deriv-com/quill-ui';
+import { getElementById } from '../../../_common/common_functions';
+import { localize } from '../../../_common/localize.js';
+import contractManager from '../../common/contract_manager.js';
+
+const NotAvailable = ({ title, body }) => (
+    <div className='not-available-container'>
+        <section className='not-available-section'>
+            <Text size='xl' bold centered color='not-available-section-text'>
+                {title}
+            </Text>
+            <Text size='lg' centered color='not-available-section-text'>
+                {body}
+            </Text>
+            <Button
+                onClick={(e) => {
+                    setTimeout(() => {
+                        document.getElementById('acc_switcher').click();
```

Comment:
why do we need setTimeout here

Reply:
There's a strange behaviour that stops the trigger when instantly fired. Adding the timeout is a dirty workaround.